cranelift/src/disasm.rs

```diff
@@ -111,9 +111,18 @@ cfg_if! {
 
         fn get_disassembler(isa: &dyn TargetIsa) -> Result<Capstone> {
             let cs = match isa.triple().architecture {
-                Architecture::Riscv32(_) | Architecture::Riscv64(_) => {
-                    anyhow::bail!("No disassembler for RiscV");
-                }
+                Architecture::Riscv32(_) => Capstone::new()
+                    .riscv()
+                    .mode(arch::riscv::ArchMode::RiscV32)
+                    .extra_mode(std::iter::once(arch::riscv::ArchExtraMode::RiscVC))
+                    .build()
+                    .map_err(map_caperr)?,
+                Architecture::Riscv64(_) => Capstone::new()
+                    .riscv()
+                    .mode(arch::riscv::ArchMode::RiscV64)
+                    .extra_mode(std::iter::once(arch::riscv::ArchExtraMode::RiscVC))
+                    .build()
+                    .map_err(map_caperr)?,
```

Comment:
I think these should use `.set_skipdata` like AArch64. That ensures that capstone prints inline constant pools as raw bytes instead of truncating the instruction stream.

Reply:
I will fix that.